src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/UserEntity.kt

```diff
@@ -316,8 +316,12 @@ enum class UserRole(val service: ServiceName, val cas1ApiValue: ApprovedPremises
   CAS1_EXCLUDED_FROM_ASSESS_ALLOCATION(ServiceName.approvedPremises, ApprovedPremisesUserRole.excludedFromAssessAllocation),
   CAS1_EXCLUDED_FROM_MATCH_ALLOCATION(ServiceName.approvedPremises, ApprovedPremisesUserRole.excludedFromMatchAllocation),
   CAS1_EXCLUDED_FROM_PLACEMENT_APPLICATION_ALLOCATION(ServiceName.approvedPremises, ApprovedPremisesUserRole.excludedFromPlacementApplicationAllocation),
-  CAS1_APPEALS_MANAGER(ServiceName.approvedPremises, ApprovedPremisesUserRole.appealsManager),
-  CAS1_JANITOR(ServiceName.approvedPremises, ApprovedPremisesUserRole.janitor),
+  CAS1_APPEALS_MANAGER(
+    ServiceName.approvedPremises,
+    ApprovedPremisesUserRole.appealsManager,
+    listOf(UserPermission.CAS1_PROCESS_AN_APPEAL, UserPermission.CAS1_VIEW_ASSESSMENTS, UserPermission.CAS1_ASSESS_APPEALED_APPLICATION),
+  ),
```

Comment:
Have you tried running through the whole process on the UI with a user that only has the CAS1_APPEALS_MANAGER role? I've made similar changes before and been caught out because there have been permission checks when submitted assessments that lead to permission errors